dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -397,7 +397,14 @@ def _batched_qr(a, mode="reduced"):
         batch_size,
         depends=[copy_ev],
     )
-    _manager.add_event_pair(ht_ev, geqrf_ev)
+
+    # w/a to avoid raice conditional on CUDA during multiple runs
+    # TODO: Remove it ones the OneMath issue is resolved
+    # https://github.com/uxlfoundation/oneMath/issues/626
+    if dpnp.is_cuda_backend(a_sycl_queue):
```

Comment:
dpctl implements an approach to omit certain branches, certain lines from coverage stats in [dpctl#1970](https://github.com/IntelPython/dpctl/pull/1970).
Should we do something similar here?
```suggestion  # pragma: no cover
    if dpnp.is_cuda_backend(a_sycl_queue):
```

Reply:
I can add this.
But I would prefer to create a separate PR to omit lines with this check from coverage report

Reply:
I don't mind to handle that separately.